lnbits/core/services.py

```diff
@@ -577,7 +577,7 @@ async def check_transaction_status(
         wallet_id, payment_hash, conn=conn
     )
     if not payment:
-        return PaymentStatus(None)
+        return PaymentPendingStatus()
     if not payment.pending:
         # note: before, we still checked the status of the payment again
         return PaymentStatus(True, fee_msat=payment.fee)
```

Comment:
Shouldn't we also change this?

```suggestion
        return PaymentSuccessStatus(fee_msat=payment.fee)
```

So we will remove all usages of `PaymentStatus` from the codebase and just use the specific classes and never the generic one?

Reply:
Done in commit: https://github.com/lnbits/lnbits/pull/2322/commits/575bc5154bc979a75cb88cc17aa76c3832338a85